src/test/java/com/cloudant/tests/DatabaseTest.java

```diff
@@ -71,6 +72,22 @@ public void permissions() {
 
     }
 
+    /**
+     * Test that when called against a DB that is not a Cloudant service
+     * an UnsupportedOperationException is thrown
+     */
+    @Test
+    public void testPermissionsException() {
+        try {
+            Map<String, EnumSet<Permissions>> userPerms = db.getPermissions();
+            //ignore the test in cases where we don't get an exception
+            Assume.assumeTrue(false);

```

Comment:
New test categories as discussed including updated running test information in: 8ea51b8 and 0679f26